spryker/usr/local/share/spryker/spryker_functions.sh

```diff
@@ -98,7 +104,7 @@ do_spryker_generate_files() {
 do_spryker_install() {
   # check if database exists (it is supposed to be created by postgres container)
   set +e
-    psql -U "$DATABASE_USER" -h "$DATABASE_HOST" -lqt | cut -d \| -f 1 | grep -q "$DATABASE_NAME"
+    psql -U "$DATABASE_USER" -h "$DATABASE_HOST" -lqt "$DATABASE_NAME" | cut -d \| -f 1 | grep -q "$DATABASE_NAME"
```

Comment:
I've not needed the database name for this before, I wonder why you do now

Reply:
Because it's `postgres` by default, which is unauthorized for non privileged PostgreSQL users.

Reply:
perhaps something specific about the permissions the postgres docker image sets up

Reply:
probably need to update the php/usr/share/assets/assets_functions.sh psql stuff too then

Reply:
It did not cause any issue so far so I'd vote to keep them as they are

Reply:
Actually, that makes sense to change them as well... doing that.